src/rotest/core/abstract_test.py

```diff
@@ -331,3 +330,146 @@ def store_state(self):
 
         for resource in self.all_resources.itervalues():
             resource.store_state(store_dir)
+
+    def _wrap_assert(self, assert_method, *args, **kwargs):
+        try:
+            assert_method(*args, **kwargs)
+
+        except AssertionError as err:
+            self.expect(False, str(err))

```

Comment:
I don't like this part, 
the `self.expect(False)` bit to be exact, would the user get a weird Error such as `False is not True`?

Reply:
Also what about the traceback?